src/routes/Connect.svelte

```diff
@@ -7,3 +12,17 @@
     <li class="simple-signer xbull-wallet">Connect with xBull</li>
     <li class="simple-signer private-key-wallet">Connect with Private Key</li>
 </ul>
+<h1>Public Key: {$publicKey ? $publicKey : 'Waiting connection...'}</h1>
+<button on:click="{() => ($isVisible = !$isVisible)}">Show key</button>
```

Comment:
try to avoid inline mutations

Reply:
add docs to svelte reference where it says this is recommended

Reply:
https://svelte.dev/tutorial/inline-handlers